ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -344,4 +346,55 @@ - (void)testUseMainForkIfNamedStream {
     vnode_put(testVnode);
 }
 
+- (void)testResizePendingRenamesRace
+{
+    InitPendingRenames();
+    XCTAssertLessThan(s_maxPendingRenames, 16);
```

Comment:
Very small nit:  If we know what the value should be lets assertEqual

Reply:
I guess as we're directly testing the data structure itself here we can do that. Otherwise I'd say we don't really care about the exact growth algorithm.